AdobeStockAsset/etc/request.xsd

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
Can we use DI configuration (arguments) to avoid introducing a new schema? What are the benefits of custom schema comparing to the DI configuration approach?